main.go

```diff
@@ -129,6 +129,9 @@ func mainError() error {
 
 	daemonCommand.PersistentFlags().Bool(f.Service.AWS.IncludeTags, true, "Should resource tags be included (especially for restricted regions, like S3 buckets in China regions).")
 
+	daemonCommand.PersistentFlags().String(f.Service.AWS.Encrypter, "kvm", "Encryption backend to use.")
+	daemonCommand.PersistentFlags().String(f.Service.AWS.VaultAddress, "", "Server address for Vault encryption.")
```

Comment:
The flags ordering was already messy but would be good to have these alphabetical.

Reply:
Sure, added to the first block in the right order.